tutorials/ai-launchpad-producer/ai-launchpad-producer.md

```diff
@@ -0,0 +1,397 @@
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 8: ](Start training with execution)]
+
+> ### What's an execution?
+>
+>Execution refers to the training process. It is a one-life process, therefore, you use the **configuration** to create another execution with the same settings (value bindings).
+
+1. Click **Execution** under **ML Operations**. Then click **Create**.
+
+    !![Execution Create](img/train-1.png)
+
+2. Select configuration named `airlines-complaint-training`.
+
+    Verify your configuration ID (may be different from show image) and click **Create**.
+
+    !![Execution Select Configuration](img/train-2.png)
+
+    You will be redirected to dashboard of the newly created **Execution**.
+
+2.	Observe each step of your training process.
+
+    !![Execution Create](img/train-3.png)
+
+3.	Click **Refresh**.
+
+    Once your execution status changes to `COMPLETED`, it will generate **Output Artifact** that is, model for text classification.
+
+    !![Execution Create](img/train-4.png)
+
+Take a note for your generated Output Artifact's (model) ID. You can always visit the execution details page to get this.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 9: ](Set deployment configuration)]
+
+Previously you bound the **Workflow Executable** to the dataset using **configuration** to train model.
+
+Now you will bind **Serving Executable** to the model using configuration to deploy the model.
+
+1. Select **Configuration** under **ML Operations**. Click **Create**.
+
+    !![Create configuration](img/config-0.png)
+
+
+2. Set the configuration details as shown below.
+
+    !![Create configuration](img/deploy-1.png)
+    <div>&nbsp;</div>
+
+    |  Field Name     | Value | Note
+    |  :------------- | :------------- | :------
+    |  Configuration Name | **`airlines-complaint-deploy`**   | |
+    |  Scenario           | **`text-clf-tutorial-scenario`**    | |
+    |  Version            | **`2.0.0`** or **`1.0.0`**          | Choose version similar to synced workflow |
+    |  Executable         | **`text-clf-infer-tutorial-exec`**  | **Serving executable** refers to an executable for deploying the model. |
+
+
+3. Click **Next** when asked for **Input Parameters**.
+    > With **Input Parameters** you set values to the placeholder for hyper-parameters in your executable.
+
+    !![Create configuration](img/config-2.png)
+
+4. Identify your model row by in the **Available Artifacts** pane using the unique ID. Select the drop-down next to it and check `text-data`.
+
+    > Input artifacts refers to placeholder for artifacts (dataset or models) in your executable. Here the `textmodel` is the placeholder name for you set the model (artifact) id.
+
+    !![Create configuration](img/deploy-3.png)
+
+5. Review your configuration settings and click **Create**.
+
+    !![Create configuration](img/deploy-4.png)
+
+You will see the overview of your configuration. The configuration ID is uniquely identifies your configuration.
```

Comment:
Missing "and"?

Reply:
Please address this issue.
"The configuration ID is uniquely identifies your configuration." is grammatically incorrect. Either add "and" between "uniquely" and "identifies", and change "uniquely" into "unique" or remove the "is".